modules/Media.js

```diff
@@ -31,7 +30,7 @@ class Media extends React.Component {
 
   componentWillMount() {
     let { query } = this.props;
-    const { targetWindow } = this.props;
+    const targetWindow = this.props.targetWindow || window;
```

Comment:
This still breaks server-side rendering because `componentWillMount` fires on the server but `window` is not defined.